app/models/manager_refresh/inventory/middleware_manager.rb

```diff
@@ -0,0 +1,62 @@
+module ManagerRefresh::Inventory::MiddlewareManager
+  extend ActiveSupport::Concern
+  include Core
+
+  COMMON_ATTRIBUTES = %i(name ems_ref nativeid properties feed).freeze
+
+  class_methods do
+    def has_middleware_manager_domains(options = {})
```

Comment:
@cfcosta Got a request to move these definitions to `ManagerRefresh` namespace to make them available outside Hawkular provider. But these names are raising rubocop warnings. Are you ok with these names? I'm trying to be consistent with these ones: https://git.io/vSBuV